dask_sql/physical/rel/logical/aggregate.py

```diff
@@ -526,8 +514,6 @@ def _perform_aggregation(
                 logger.debug(f"Using original output_col value of '{output_col}'")
 
             aggregations_dict[input_col][output_col] = aggregation_f
-            if not isinstance(aggregation_f, str):
-                fast_groupby = False
```

Comment:
I initially included this because we were seeing groupby test failures when using `dd.Aggregations` without our custom null sorting, but it seems like this is no longer the case?

Reply:
I'm assuming that there are tests with nulls checking for dd.Aggregation type of aggregations but it might be worth verifying before adding this in.

Reply:
We actually have a lot more exposure for `dd.Aggregations` now with https://github.com/dask-contrib/dask-sql/pull/823 since that introduced a `custom_sum` aggregation in the place of standard `sum` aggregations on CPU; one test that is using this on a dataframe with nulls is `test_single_agg_count_no_group_by` (along with a few other tests in test_compatibility.py).